### DISCUSSION TITLE
Curation report Jan 9 - 13

### DISCUSSION THREAD

**Discussion by elissonnog** [wikipathways/GPML-Repo#107] [Curation reports]
Hello Team, 

I m late, sorry.

Here is my report:

- Approved tags:
WP4346, WP4347,WP462, WP266, WP1269, WP2634

- Request deletion:
WP5291, 

- Needs Work:
WP5305, WP5311, WP5312, WP5314, WP5315, 

- Incomplete:
WP5293, 

- Add References: 
WP5314

- Fix Interactions:
WP5313, WP5311

- Annotate nodes:
WP5290, 

- Assigned as lipid
WP5304,

Ps:  This morning, servers were very slow to response and crashed a few times when a tag was added.

Search works fine.

Best regards,
Elisson

**Comment by khanspers** [comment]
Hi Elisson, thanks for your report. For WP5293, I am still working on finalizing it and hope to get to that this week.

I also noticed the site being slow this morning, but it seems to be back to normal now.

Kristina
